Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM node:12-alpine
+
+RUN mkdir -p /usr/app
+WORKDIR /usr/app
+
+COPY package.json yarn.lock tsconfig.json tsconfig.build.json start.sh ./
+COPY src ./src
+COPY public ./public
+COPY views ./views
+
+ADD https://github.com/ufoscout/docker-compose-wait/releases/download/2.2.1/wait /wait
```

Comment:
why do you use docker-compose-wait?

Reply:
It's a practice used to ensure the prerequisite services (here, the postgresql db) are ready to accept connections before we start our app